reference/configuration/framework.rst

```diff
@@ -526,10 +526,10 @@ enabled
 
 Whether or not to enable the ``translator`` service in the service container.
 
-fallback
-........
+fallbacks
+.........
 
-**type**: ``string`` **default**: ``en``
+**type**: ``array`` **default**: ``en``
```

Comment:
Ah, sorry. I missed that, in that case I think you should still add a versionadded (since it still is new behaviour), but then using `2.3.x` (where x is the first release version it is included in (unless @xabbuh or others disagree with me)